src/types/TodoFilter.ts

```diff
@@ -0,0 +1,6 @@
+import { Status } from './Status';
+
+export type TodoFilterContextType = {
```

Comment:
```suggestion
export interface TodoFilterContextType  {
```
use interface prefer